enkit/outputs/configuration/README.md

```diff
@@ -0,0 +1,7 @@
+### Updating the configs to astore:
+> assuming you're using the default paths, running something like 
+```
+    enkit astore upload realconfig.yaml@enkitconfigs/outputs.yaml
+```
+
+will update the prefilled configs for the command 
```

Comment:
How will someone know when this needs to be done?

Let's make updating the config an explicit list of steps, like:

```
1. Download the current config: `insert command here`
2. Change paramenters in the downloaded config
3. Upload modified config back to astore: `insert command here`
4. Set `live` tag in astore (or whatever else needs to be done - command for each)
```

I'm thinking that providing a blank template gives people space to fill in the wrong parameters because they know e.g. the new API key but in the process of creating a new config, mess up the cluster name

...and then make sure there's a link here in the relevant place (playbook for enkit?) or just inline this in that one place